integration/src/test/java/org/kie/hacep/KafkaUtilTest.java

```diff
@@ -194,7 +195,7 @@ public void createTopics(String... topics) {
                 }
             }
         } catch (Exception e) {
-            e.printStackTrace();
+            log.error(e.getMessage(), e);
```

Comment:
I think it will be better to rethrow exception and fail fast in this case. Stack trace will be printed by surefire.
Something like:
throw new RuntimeException(e.getMessage(), e);

Reply:
ok